src/test/java/com/faforever/client/map/MapVaultControllerTest.java

```diff
@@ -159,7 +159,7 @@ public void testNotifyPropertyShowLadderInitialized() {
     instance.display(new ShowLadderMapsEvent());
 
     WaitForAsyncUtils.waitForFxEvents();
-    verify(mapService).getLadderMaps(100, 1);
+    verify(mapService).getLadderMaps(50, 1);
   }
```

Comment:
Maybe add a unit test that tests if pagination works

Reply:
Ok i will look into that.